src/commands/integrations/flows/test.ts

```diff
@@ -266,7 +266,7 @@ prism integrations:flows:test -u=${invokeUrl} ${flagString}
 
     const executionId = response.headers["prismatic-executionid"];
 
-    if (!response.data.executionId) {
+    if (!response.data || !response.data.executionId) {
```

Comment:
Nit but perhaps `!response?.data?.executionId`? 

Also line 267 is grabbing the `executionId` from the headers (should work for async and sync both) but this only cares about the async behavior of including it in the response body.

Reply:
Yeah this is a bit of a silly workaround but IIRC we don't currently have a good way of telling if a flow was run sync or async (unless there's something jammed in the header I don't know about yet) aside from the fact that async flows always return executionId's in the body.